src/plugins/embeddable/public/lib/panel/panel_header/panel_header.tsx

```diff
@@ -151,6 +151,21 @@ export function PanelHeader({
       </span>
     );
   };
+  
+  if (hidePanelTitle){
+    return (
+      <div data-test-subj="dashboardPanelTitle__wrapper" className={classes}>
+        <PanelOptionsMenu
+          getActionContextMenuPanel={getActionContextMenuPanel}
+          isViewMode={!isViewMode}
```

Comment:
This is not an ideal solution:
1. Lots of code duplication - do we really need to duplicate the entire `dashboardPanelTitle__wrapper` for either case?
2. The logic on why this works isn't clear - could we restructure this code to make the logic more clear?
3. A better solution would be to conditionally render the `dashboardPanelTitle__wrapper` **without** changing the logic of the `PanelOptionsMenu`

Reply:
Ok, Thank you. 
I work on this one, and ask for a review.